llvm/include/llvm/Analysis/TargetTransformInfoImpl.h

```diff
@@ -189,27 +189,28 @@ class TargetTransformInfoImplBase {
 
     // These will all likely lower to a single selection DAG node.
     // clang-format off
-    if (Name == "copysign" || Name == "copysignf" || Name == "copysignl" ||
-        Name == "fabs"  || Name == "fabsf"  || Name == "fabsl" ||
-        Name == "fmin"  || Name == "fminf"  || Name == "fminl" ||
-        Name == "fmax"  || Name == "fmaxf"  || Name == "fmaxl" ||
-        Name == "sin"   || Name == "sinf"   || Name == "sinl"  ||
-        Name == "cos"   || Name == "cosf"   || Name == "cosl"  ||
-        Name == "tan"   || Name == "tanf"   || Name == "tanl"  ||
-        Name == "asin"  || Name == "asinf"  || Name == "asinl" ||
-        Name == "acos"  || Name == "acosf"  || Name == "acosl" ||
-        Name == "atan"  || Name == "atanf"  || Name == "atanl" ||
-        Name == "atan2" || Name == "atan2f" || Name == "atan2l"||
-        Name == "sinh"  || Name == "sinhf"  || Name == "sinhl" ||
-        Name == "cosh"  || Name == "coshf"  || Name == "coshl" ||
-        Name == "tanh"  || Name == "tanhf"  || Name == "tanhl" ||
-        Name == "sqrt"  || Name == "sqrtf"  || Name == "sqrtl" ||
-        Name == "exp10"  || Name == "exp10l"  || Name == "exp10f")
+    if (Name == "copysign" || Name == "copysignf" || Name == "copysignl" || Name == "copysignf128" ||
+        Name == "fabs"  || Name == "fabsf"  || Name == "fabsl"  || Name  == "fabsf128" ||
+        Name == "fmin"  || Name == "fminf"  || Name == "fminl"  || Name  == "fminf128" ||
+        Name == "fmax"  || Name == "fmaxf"  || Name == "fmaxl"  || Name  == "fmaxf128" ||
+        Name == "sin"   || Name == "sinf"   || Name == "sinl"   || Name  == "sinf128"  ||
+        Name == "cos"   || Name == "cosf"   || Name == "cosl"   || Name  == "cosf128"  ||
+        Name == "tan"   || Name == "tanf"   || Name == "tanl"   || Name  == "tanf128"  ||
+        Name == "asin"  || Name == "asinf"  || Name == "asinl"  || Name  == "asinf128" ||
+        Name == "acos"  || Name == "acosf"  || Name == "acosl"  || Name  == "acosf128" ||
+        Name == "atan"  || Name == "atanf"  || Name == "atanl"  || Name  == "atanf128" ||
+        Name == "atan2" || Name == "atan2f" || Name == "atan2l" || Name  == "atan2f128"||
+        Name == "sinh"  || Name == "sinhf"  || Name == "sinhl"  || Name  == "sinhf128" ||
+        Name == "cosh"  || Name == "coshf"  || Name == "coshl"  || Name  == "coshf128" ||
+        Name == "tanh"  || Name == "tanhf"  || Name == "tanhl"  || Name  == "tanhf128" ||
+        Name == "sqrt"  || Name == "sqrtf"  || Name == "sqrtl"  || Name  == "sqrtf128" ||
+        Name == "exp10" || Name == "exp10l" || Name == "exp10f" || Name  == "exp10f128")
```

Comment:
I'm not sure what the effects of this change are, but it seemed unusual that long double and f128 would be treated differently here.